src/test/java/model/base/BasePage.java

```diff
@@ -22,4 +49,30 @@ public EditPageType clickNewRecordButton() {
 
         return createEditPage();
     }
+
+    public String getCardBodyText() {
+
+        return cardBody.getText();
+    }
+
+    public ViewPageType clickMenuView() {
+        clickActionsButton();
+        getWait().until(ExpectedConditions.elementToBeClickable(viewButton)).click();
+
+        return createViewPage();
+    }
+
+    public EditPageType clickMenuEdit() {
+        clickActionsButton();
+        getWait().until(ExpectedConditions.elementToBeClickable(editButton)).click();
+
+        return createEditPage();
+    }
+
+    public MainPageType clickMenuDelete() {
+        clickActionsButton();
+        getWait().until(ExpectedConditions.elementToBeClickable(deleteButton)).click();
+
+        return createMainPage();
```

Comment:
Тут обязательно пересоздавать страницу, что если вернуть себя же?
```suggestion
        return (MainPageType) this;
```